plenum/common/messages/node_messages.py

```diff
@@ -282,10 +281,11 @@ class MessageReq(MessageBase):
     """
     Purpose: ask node for any message
     """
+    allowed_types = {LEDGER_STATUS, CONSISTENCY_PROOF, PREPREPARE,
```

Comment:
There are tests present in `plenum/test/node_request/message_request/test_valid_message_request.py`